aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -461,14 +462,35 @@ Tensor view_dtype(const Tensor& self, ScalarType dtype) {
   return new_tensor;
 }
 
+// Sparse layout conversions Start
+
 Tensor dense_to_sparse_csr(const Tensor& self) {
   return self.to_sparse().to_sparse_csr();
 }
 
+Tensor dense_to_sparse_csc(const Tensor& self) {
+  AT_ERROR(
+      "Conversion from ", self.layout(), " to CSC is currently not supported.");
```

Comment:
Nit: Notice that `self.layout()` is displayed as "SparseCsr" if the self layout is CSR. Hence, to be consistent with naming layouts in the exception message, in principle, we have two possibilities:
```suggestion
      "Conversion from ", self.layout(), " to SparseCsc is currently not supported.");
```
or
```suggestion
      "Conversion from ", layoutToString(self.layout(), true) , " to CSC is currently not supported.");
```
Since `layoutToString` does not support non-sparse layouts (perhaps it should), we are left with the first choice.

However, this is a non-blocking issue considering that we'll need to implement the conversion methods anyway.

The same comment applies to other exception messages as well.